ote_sdk/ote_sdk/usecases/exportable_code/demo/README.md

```diff
@@ -5,54 +5,90 @@ Name of package is a name of model which was deployed.
 
 ## Structure of generated package:
 
-openvino.zip:
-
-  * [model](../model)
-    * [model.xml](../model/model.xml)
-    * [model.bin](../model/model.bin)
-  * [python](.)
-    * [README.md](./README.md)
-    * [demo.py](./demo.py)
-    * [requirements.txt](./requirements.txt)
-    * <model_name>-0.0-py3-none-any.whl
+* [model](../model)
+  * [model.xml](../model/model.xml)
+  * [model.bin](../model/model.bin)
+* [python](.)
+  * [README.md](./README.md)
+  * [demo.py](./demo.py)
+  * [requirements.txt](./requirements.txt)
+  * [demo_package-0.0-py3-none-any.whl](./demo_package-0.0-py3-none-any.whl)
 
 
 ## Prerequisites
-* Python 3.6+
+* Python 3.8+
 
 ## Setup Demo Package
 
-1. Install Python (version 3.6 or higher).
+1. Install Python (version 3.8 or higher).
 
 2. Install the package in the clean environment:
 ```
-python -m pip install <model_name>-0.0-py3-none-any.whl
+python -m pip install demo_package-0.0-py3-none-any.whl
 ```
 
 
 When the package is installed, you can import it as follows:
 ```
-python -c "from <model_name> import create_model"
+python -c "from demo_package import create_model"
```

Comment:
Let's keep it out of this PRs scope, but moving forward, how multiple models are supposed to be used in the same environment?

Reply:
Now I don't drop package (zip) in deploy. So name of zip will specified when developer save archive after deploy. Also wheel is independent from model (except for the `config.json`) So maybe moving `config` to model directory make sense

Reply:
What's the purpose of a package then?

Reply:
For running inference for model

Reply:
I'd propose to merge this PR ASAP, but for the backlog:

1. If the name of a package does not depend on model, it's type or whatever, you can install multiple packages in the same environment. This may be a significant drawback for task chaining support.
2. Currently there main components of a package are: config, utils to create model and output converter, sync demo pipeline. Default sync demo pipeline can be moved to OTE SDK (or even to Model API), because it does not contain any model specific logic. Default implementations of utils just map parameters from config file to classes in OTE SDK, and thus can be moved to OTE SDK. Config is the only thing solely dependent on the model. But there is no particular need to create a Python package just to put config there. To summarize, it looks like we need (non-dummy) package only for non-default pipelines and model wrappers, otherwise we can just use the SDK. What do you think?